### PR TITLE
Support WRED profiles on system ports

### PR DESCRIPTION
**Pull request by vmittal-msft**
**What I did**
Added support to apply wred profile on system ports

**Why I did it**
remote system ports were missing wred profile

**How I verified it**
Tested on sonic chassis

**Details if related**

### CODE REVIEW

**Comment by vmittal-msft** [comment]
@cheryls-arista please review. This is un-doing the fix you did for Inband ports. Your fix caused issue while applying buffer queue profile on system ports. We can discuss further,

**Comment by vmittal-msft** [comment]
This is the one i am mentioning about - https://github.com/sonic-net/sonic-swss/pull/2740

**Comment by vmittal-msft** [comment]
Template change PR - https://github.com/sonic-net/sonic-buildimage/pull/15612

**Comment by vmittal-msft** [comment]
@cheryls-arista After your fix, i started getting this error for local ports . My PR fixes this issue.
image

File: orchagent/portsorch.cpp [hunk 1]
```diff
@@ -7971,8 +7971,8 @@ bool PortsOrch::addSystemPorts()
                     }
                 }
 
-                //System port for local port. Update the system port info in the existing physical port
-                if(!getPort(attr.value.oid, port))
+                Port local_port;
```
Comment (kenneth-arista) [inline orchagent/portsorch.cpp:7974]: As discussed yesterday, this change will break inband ports in Voq chassis.

**Comment by vmittal-msft** [comment]
Opened new PR - https://github.com/sonic-net/sonic-swss/pull/2851 with latest change. hence closing this now.

**Pull request closed without merging**